discovery/client.go

```diff
@@ -52,6 +52,8 @@ type clientRegistrationManager interface {
 	validate() error
 	// removeRevoked removes all revoked presentations from the store
 	removeRevoked() error
+	// getServiceAndSubject returns the service and subject, or ErrServiceNotFound / didsubject.ErrSubjectNotFound if either does not exist
+	getServiceAndSubject(ctx context.Context, serviceID, subjectID string) (ServiceDefinition, []did.DID, error)
```

Comment:
why add it to the interface? There's no mock and only use is at the same struct.

Reply:
removed interface since it only had 1 implementation